delivery_auto_refresh/models/sale_order.py

```diff
@@ -18,8 +16,13 @@ class SaleOrder(models.Model):
     @api.onchange("partner_id", "partner_shipping_id")
     def _onchange_partner_id(self):
         if hasattr(super(), "_onchange_partner_id"):
-            super()._onchange_partner_id()
-        if get_bool_param(self.env, "set_default_carrier"):
+            return super()._onchange_partner_id()
+        default_carrier = (
+            self.env["ir.config_parameter"]
+            .sudo()
+            .get_param("delivery_auto_refresh.set_default_carrier")
+        )
+        if default_carrier:
```

Comment:
Let's name these variables after their params
```suggestion
        set_default_carrier = (
            self.env["ir.config_parameter"]
            .sudo()
            .get_param("delivery_auto_refresh.set_default_carrier")
        )
        if set_default_carrier:
```